catalog.html

```diff
@@ -1,118 +1,112 @@
+        </ul>
+    </nav>
 </header>
-<main>
-  <section>
-    <h2></h2>
-    Инфраструктура
-    Бассейн
-    Парковка
-    Wi-Fi
-    Тип Жилья
-    Гостиница
-    Мотель
-    Апартаменты
-    Стоимость в сутки(Р)
-    от 0 до 3000
-    Показать
-  </section>
-  <section>
-    <h2></h2>
-    Найдено: 3
-    Сортировка:
-    <ul>
-      <li><a href="">По цене</a></li>
-      <li><a href="">По типу</a></li>
-      <li><a href="">По рейтингу</a></li>
-    </ul>
-    <ul>
-      <li>
-        <article>
-          <h3><a href="#!">Amara resort & spa</a></h3>
-          <a href="">
-            <img src="" alt="">
-          </a>
-          <span>Гостиница</span>
-          <span>от 4000 Р</span>
-          <div class=""></div>
-          <!--рейтинг в звёздах-->
-          <p>Рейтинг:8.5</p>
-          <button type="button">Подробнее</button>
-          <button type="button">Забронировать</button>
-        </article>
-      </li>
-      <li>
-        <article>
-          <h3><a href="#!">Desert Quail inn</a></h3>
-          <a href="">
-            <img src="" alt="">
-          </a>
-          <span>Мотель</span>
-          <span>от 3000 Р</span>
-          <p>Рейтинг:8.9</p>
-          <div class=""></div>
-          <!--рейтинг в звёздах-->
-          <button type="button">Подробнее</button>
-          <button type="button">Забронировать</button>
-        </article>
-      </li>
-      <li>
-        <article>
-          <h3><a href="#!">Villas at poco diablo</a></h3>
-          <a href="">
-            <img src="" alt="">
-          </a>
-          <span>Апартаменты</span>
-          <span>от 2000 Р</span>
-          <div class=""></div>
-          <!--рейтинг в звёздах-->
-          <p>Рейтинг:9.2</p>
-          <button type="button">Подробнее</button>
-          <button type="button">Забронировать</button>
-        </article>
-      </li>
-    </ul>
-  </section>
+<main class="container">
+    <section class="propetries">
+        <h2></h2>
+        Инфраструктура Бассейн Парковка Wi-Fi Тип Жилья Гостиница Мотель Апартаменты Стоимость в сутки(Р) от 0 до 3000 Показать
+    </section>
+    <section>
+        <h2></h2>
+        <span class="results">Найдено:3</span>
+        <p class="definition">Сортировка:</p>
+        <ul class="category list-reset">
+            <li><a class="price" href="">По цене</a></li>
+            <li><a class="type" href="">По типу</a></li>
+            <li><a class="rate" href="">По рейтингу</a></li>
+            <li><a class="up" href="">По возрастанию</a></li>
+            <li><a class="down" href="">По убыванию</a></li>
+        </ul>
+        <ul class="search-results list-reset">
+            <li class="search-item">
+                <article>
+                    <h3 class="item-title"><a href="#!">Amara resort & spa</a></h3>
+                    <a href="">
+                        <img src="" alt="">
+                    </a>
+                    <span class="item-type">Гостиница</span>
```

Comment:
обычно такие метки называют тегами (не html):
```suggestion
                    <span class="hotel-card-tag">Гостиница</span>
```